Do not override PYTHONPATH when calling Python code from the Haskell
daemons.  This is necessary because the Python library dependencies are
only available through PYTHONPATH.

diff --git a/src/Ganeti/Query/Exec.hs b/src/Ganeti/Query/Exec.hs
--- a/src/Ganeti/Query/Exec.hs
+++ b/src/Ganeti/Query/Exec.hs
@@ -99,12 +99,10 @@ spawnJobProcess jid = withErrorLogAt CRITICAL (show jid) $
   do
     use_debug <- isDebugMode
     env_ <- (M.toList . M.insert "GNT_DEBUG" (if use_debug then "1" else "0")
-            . M.insert "PYTHONPATH" AC.pythondir
             . M.fromList)
            `liftM` getEnvironment
     execPy <- P.jqueueExecutorPy
     logDebug $ "Executing " ++ AC.pythonPath ++ " " ++ execPy
-               ++ " with PYTHONPATH=" ++ AC.pythondir
 
     (master, child) <- pipeClient connectConfig
     let (rh, wh) = clientToHandle child

